src/main/java/org/cbioportal/cdd/service/internal/ClinicalAttributeMetadataCache.java

```diff
@@ -124,6 +124,7 @@ public void resetCache(boolean force) {
         for (Map.Entry<String, ArrayList<ClinicalAttributeMetadata>> entry : latestOverrides.entrySet()) {
             HashMap<String, ClinicalAttributeMetadata> clinicalAttributesMetadataMapping = new HashMap<String, ClinicalAttributeMetadata>();
             for (ClinicalAttributeMetadata clinicalAttributeMetadata : entry.getValue()) {
+                fillOverrideAttributeWithDefaultValues(clinicalAttributeMetadata, latestClinicalAttributeMetadataCache.get(clinicalAttributeMetadata.getColumnHeader()));
```

Comment:
Note we should check if the google repository has an empty value, if that is null or empty string.  If we wanted a null value to overwrite a default value, this will not work.

Reply:
Maybe this should be put into the TopBraid repository instead, so that the Google and TopBraid repositories return the same thing.

Reply:
I think the google representation of overrides was preliminary and it is ok to leave that functionality behind. In the new representation I think we can understand that it is up to the curator of overrides to adjust overrides appropriately any time the default values for an attribute change. If they wish to "lock in" a particular value in the override for all time and to not float along with the default values and how they may change in the future, they can do so by adding the OverrideValue records for all explicitly overridden metadata attributes.